### ISSUE TITLE
Failed to create graphics device!

### ISSUE THREAD

**Issue by Blizzardo1** [Thraka/SadConsole#130]
Microsoft.Xna.Framework.Graphics.NoSuitableGraphicsDeviceException
  HResult=0x80131500
  Message=Failed to create graphics device!
  Source=MonoGame.Framework
  StackTrace:
   at Microsoft.Xna.Framework.GraphicsDeviceManager.CreateDevice()
   at Microsoft.Xna.Framework.Game.DoInitialize()
   at Microsoft.Xna.Framework.Game.Run(GameRunBehavior runBehavior)
   at Term.Program.Main(String[] args)

Inner Exception 1:
EntryPointNotFoundException: Unable to find an entry point named 'SDL_SetWindowResizable' in DLL 'SDL2.dll'.

**Comment by Thraka** [comment]
Please read https://github.com/Thraka/SadConsole/wiki/NuGet-Starter-MonoGame#setup-sadconsole and see if the area about SDL2 solves your problem. Let me know!

**Comment by Blizzardo1** [comment]
It's still throwing
> EntryPointNotFoundException: Unable to find an entry point named 'SDL_SetWindowResizable' in DLL 'SDL2.dll'.

**Comment by Thraka** [comment]
What version of Monogame are you using from NuGet? I wonder if it got updated and now there is a different version of SDL?

**Comment by Blizzardo1** [comment]
DLL  Version is 3.6.0.1625 whereas the Runtime Version is v4.0.30319

**Comment by Thraka** [comment]
your OS?

**Comment by Thraka** [comment]
Please also verify that the DLL is in the output folder too. Barring anything odd, perhaps you should zip up your whole project and send it to me? Unless your on Mac then it may actually be something different.

**Comment by Blizzardo1** [comment]
Operating System: Windows 10 1709
I will always build my .NET projects on Windows on my Windows PC so-as-long as I live. the DLL is indeed 
in the output folder as it should be.

**Comment by Thraka** [comment]
:) Either way there is a problem with that DLL (wrong x64 vs x86?) If you want to zip up your project and send it to me, I can help investigate further, but the error is about how monogame can't load that DLL correctly for some reason.

What hardware are you using?

**Comment by Blizzardo1** [comment]
old stuff, but I've been able to run SDL before in C++ and C#. I've even recreated Stones which is in one of my repos, however Rules are broken, but it's SDL

**Comment by Thraka** [comment]
1. Which version of SadConsole are you using?
2. Can you try and add `SadConsole.Settings.GraphicsProfile = Microsoft.Xna.Framework.Graphics.GraphicsProfile.HiDef;` as the first line in your main entry point?
3. Can you run this project OK? [StarterMonoGL.zip](https://github.com/Thraka/SadConsole/files/1965444/StarterMonoGL.zip) run the **StarterProject.MonoGL.exe** file.

**Comment by Blizzardo1** [comment]
The StarterMonoGL project starts, and even though I've added the line of code to my Entry Point, It still cries for the error. I'm thinking I'm using either a bad copy of SDL, or a version of SDL that's not liked by your project. I'll have to further investigate on my end. In the meantime, I'll still have this issue open. Thank you

**Comment by Blizzardo1** [comment]
No matter what I do... I'll always get that error. I'm thinking it's an SDL library in my System's Directory I've installed way back that was never updated, even if I used the latest version, it will throw that very same error. only your program worked, and I can't even begin to explain it. I guess I'll just close out as my system is too stupid to even acknowledge the existance of SDL2 in my project's bin dir. Thank you for your help anyways.
